_data/authors.yml

```diff
@@ -2,40 +2,19 @@
 
```

Comment:
Is `authors.yml` still used at all?

Reply:
Yes, for author.github, author.twitter and author.name. Example: https://bitcoincore.org/en/2015/12/14/segregated-witness/

I'll remove that in a follow-up.

Reply:
Thanks!